### PR TITLE
Make switch-to-shakapacker build assets successfully with bootstrap variant [WIP]

### PR DESCRIPTION
**Pull request by eoinkelly**

### CODE REVIEW

File: template.rb [hunk 1]
```diff
@@ -183,7 +183,7 @@ def cleanup_package_json
   package_json["engines"] = build_engines_field
 
   # ensure that all dependency constraints are normalized
-  %w[dependencies devDependencies].each { |k| package_json[k] = normalize_dependency_constraints(package_json[k]) }
+  # %w[dependencies devDependencies].each { |k| package_json[k] = normalize_dependency_constraints(package_json[k]) }
```
Comment (eoinkelly) [inline template.rb:186]: Normalizing the deps to always start with `^` breaks the exact dep we need currently need to specify for bootstrap

Comment (eoinkelly) [reply template.rb:186]: @G-Rath This is just roughly commented out but I'm thinking maybe doing dependency normalisation might cause more problems than it solves?

**Comment by G-Rath** [comment]
@eoinkelly I've already got this fixed in #307 :)

File: variants/backend-base/postcss.config.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    require('postcss-import'),
+    require('postcss-flexbugs-fixes'),
+    require('postcss-preset-env')({
+      autoprefixer: {
+        flexbox: 'no-2009'
+      },
+      stage: 3
+    })
+  ]
+};
```
Comment (eoinkelly) [inline variants/backend-base/postcss.config.js:15]: @G-Rath I did hit errors completing a `bundle exec rails webpacker:compile` in the bootstrap variant without this file.

**Comment by eoinkelly** [comment]
This has served it's purpose.

**Pull request closed without merging**